ts/Series/Treegraph/TreegraphPoint.ts

```diff
@@ -0,0 +1,205 @@
+/* *
+ *
+ *  (c) 2010-2022 Pawel Lysy Grzegorz Blachlinski
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type TreegraphPointOptions from './TreegraphPointOptions';
+import type TreegraphNode from './TreegraphNode';
+import type TreegraphLink from './TreegraphLink';
+import type SVGElement from '../../Core/Renderer/SVG/SVGElement';
+import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
+import TreegraphSeries from './TreegraphSeries';
+import U from '../../Core/Utilities.js';
+import { CollapseButtonOptions } from './TreegraphSeriesOptions';
+import Point from '../../Core/Series/Point.js';
+import { Palette } from '../../Core/Color/Palettes';
+const { merge, addEvent, pick, fireEvent } = U;
+const {
+    seriesTypes: {
+        treemap: {
+            prototype: {
+                pointClass: TreemapPoint
+            }
+        }
+    }
+} = SeriesRegistry;
+
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+interface ShapeArgs {
+    x: number;
+    y: number;
+    height: number;
+    width: number;
+}
+
+class TreegraphPoint extends TreemapPoint {
+    public options: TreegraphPointOptions = void 0 as any;
+    public isLink = false;
+    public collapseButton?: SVGElement;
+    public series: TreegraphSeries = void 0 as any;
+    public collapsed?: boolean;
+    public node: TreegraphNode.Node = void 0 as any;
+    public level?: number;
+    public linkToParent?: TreegraphLink;
+
+    draw(): void {
```

Comment:
- Functions section comment missing
- public keyword for class functions are missing